NOTICE.txt

```diff
@@ -216,3 +216,207 @@ In addition, certain files include the notices provided below.
 // shall not be used in advertising or otherwise to promote the sale,
 // use or other dealings in these Data Files or Software without prior
 // written authorization of the copyright holder.
+
+----------------------
+
+                                 Apache License

```

Comment:
If we're using `google/benchmark` purely as a submodule, without copying code from it into the `microsoft/STL` repo proper, then my understanding is that we should not update `NOTICE.txt`.